### PR TITLE
Update parameters with recent changes

### PR DESCRIPTION
**Pull request by nohwnd**
## Summary

Changed description to hint more of the current possibilities while hopefully remaining to the point. 

Updated descriptions of some of the parameters that seemed misleading or not descriptive enough.

Related https://github.com/dotnet/docs/issues/15524

### CODE REVIEW

**Comment by nohwnd** [comment]
The docs content validation fails with a warning that points to a page that is 404. https://aka.ms/LocaleSpecificLinks Please advice on how to fix.

File: docs/core/tools/dotnet-test.md [hunk 1]
```diff
@@ -31,31 +31,37 @@ dotnet test -h|--help
...
+The `dotnet test` command is used to execute unit tests in a given solution. The `dotnet test` command builds the solution and runs a test host application for each test project in the solution. The test host executes tests in the given project using a test framework, for example: MSTest, NUnit, or xUnit, and reports the success or failure of each test. If all tests are successful, the test runner returns 0 as an exit code; otherwise if any test fails, it returns 1.
+
+For multi-targeted projects, tests are run for each targeted framework. The test host and the unit test framework are packaged as NuGet packages and are restored as ordinary dependencies for the project.
 
 Test projects specify the test runner using an ordinary `<PackageReference>` element, as seen in the following sample project file:
 
 [!code-xml[XUnit Basic Template](../../../samples/snippets/csharp/xunit-test/xunit-test.csproj)]
 
+Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework. And `xunit.runner.visualstudio` is a test adapter, which allows xUnit framework to work with the test host.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:42]:
```suggestion
Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework. And `xunit.runner.visualstudio` is a test adapter, which allows the xUnit framework to work with the test host.
```

File: docs/core/tools/dotnet-test.md [hunk 2]
```diff
@@ -31,31 +31,37 @@ dotnet test -h|--help
...
 
 ## Arguments
 
-- **`PROJECT | SOLUTION`**
+- **`PROJECT | SOLUTION | DIRECTORY | DLL`**
 
-  Path to the test project or solution. If not specified, it defaults to current directory.
+  Path to the test project or solution. If not specified, it searches for a project or a solution in the current directory.
+  Or path to a directory that contains a project or a solution.
+  Or path to a test project dll.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:54]:
```suggestion
  - Path to the test project.
  - Path to the solution. 
  - Path to a directory that contains a project or a solution.
  - Path to a test project *.dll* file.

  If not specified, it searches for a project or a solution in the current directory.
```

File: docs/core/tools/dotnet-test.md [hunk 3]
```diff
@@ -31,31 +31,37 @@ dotnet test -h|--help
...
 
 ## Options
 
 - **`-a|--test-adapter-path <PATH_TO_ADAPTER>`**
 
-  Use the custom test adapters from the specified path in the test run.
+  Path to a directory to be searched for additional test adapters. Only dlls with suffix `.TestAdapter.dll` are inspected. If not specified the directory of the test dll is searched.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:60]:
```suggestion
  Path to a directory to be searched for additional test adapters. Only *.dll* files with suffix `.TestAdapter.dll` are inspected. If not specified, the directory of the test *.dll* is searched.
```

File: docs/core/tools/dotnet-test.md [hunk 4]
```diff
@@ -31,31 +31,37 @@ dotnet test -h|--help
...
 
 - **`--blame`**
 
-  Runs the tests in blame mode. This option is helpful in isolating problematic tests that cause the test host to crash. It creates an output file in the current directory as *Sequence.xml* that captures the order of tests execution before the crash.
+  Runs the tests in blame mode. This option is helpful in isolating problematic tests that cause the test host to crash. When crash is detected, it creates an sequence file in `TestResults/<Guid>/<Guid>_Sequence.xml` that captures the order of tests execution before the crash.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:64]:
```suggestion
  Runs the tests in blame mode. This option is helpful in isolating problematic tests that cause the test host to crash. When a crash is detected, it creates an sequence file in `TestResults/<Guid>/<Guid>_Sequence.xml` that captures the order of tests that were run before the crash.
```

File: docs/core/tools/dotnet-test.md [hunk 5]
```diff
@@ -67,11 +73,11 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
 
 - **`-d|--diag <PATH_TO_DIAGNOSTICS_FILE>`**
 
-  Enables diagnostic mode for the test platform and writes diagnostic messages to the specified file.
+  Enables diagnostic mode for the test platform and writes diagnostic messages to the specified file, and files next to it, based on the process that is logging the messages, such as `*.host_<date>.txt` for test host log, and `*.datacollector_<date>.txt` for data collector log.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:76]:
```suggestion
  Enables diagnostic mode for the test platform and writes diagnostic messages to the specified file and to files next to it. The process that is logging the messages determines which files are created, such as `*.host_<date>.txt` for test host log, and `*.datacollector_<date>.txt` for data collector log.
```

File: docs/core/tools/dotnet-test.md [hunk 6]
```diff
@@ -67,11 +73,11 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
...
 
 - **`-f|--framework <FRAMEWORK>`**
 
-  Looks for test binaries for a specific [framework](../../standard/frameworks.md).
+  Forces the use of dotnet or .NET Framework test host for the test binaries. This only determines which type of host to use. The actual framework version to be used is determined by the runtimeconfig.json of the test project. When not specified [TargetFramework assembly attribute](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.versioning.targetframeworkattribute) is used to determine the the type of host. When that attribute is stripped from the dll, .NET Framework host is used.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:80]:
```suggestion
  Forces the use of `dotnet` or .NET Framework test host for the test binaries. This option only determines which type of host to use. The actual framework version to be used is determined by the *runtimeconfig.json* of the test project. When not specified, the [TargetFramework assembly attribute](/dotnet/api/system.runtime.versioning.targetframeworkattribute) is used to determine the type of host. When that attribute is stripped from the *.dll*, the .NET Framework host is used.
```

File: docs/core/tools/dotnet-test.md [hunk 7]
```diff
@@ -130,7 +136,7 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
 
 - **`RunSettings`** arguments
 
-  Arguments are passed as `RunSettings` configurations for the test. Arguments are specified as `[name]=[value]` pairs after "-- " (note the space after --). A space is used to separate multiple `[name]=[value]` pairs.
+ Inline `RunSettings` are passed as the last arguments on the command line after after "-- " (note the space after --). Inline `RunSettings` are specified as `[name]=[value]` pairs. A space is used to separate multiple `[name]=[value]` pairs.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:139]:
```suggestion
 Inline `RunSettings` are passed as the last arguments on the command line after "-- " (note the space after --). Inline `RunSettings` are specified as `[name]=[value]` pairs. A space is used to separate multiple `[name]=[value]` pairs.
```

File: docs/core/tools/dotnet-test.md [hunk 8]
```diff
@@ -161,6 +167,12 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
   ```dotnetcli
   dotnet test --logger "console;verbosity=detailed"
   ```
+  
+  - Run the tests in the project in the current directory, and report tests that were in progress when test host crashed:
```
Comment (tdykstra) [inline docs/core/tools/dotnet-test.md:171]:
```suggestion
  - Run the tests in the project in the current directory, and report tests that were in progress when the test host crashed:
```

**Review by tdykstra — APPROVED** [review]
Looks good, I made some suggestions mainly for site style issues.

**Comment by nohwnd** [comment]
Awesome thanks! 👏

**Pull request merged**